### DISCUSSION TITLE
Mountain duck "enable finder extension" warning when enabled?

### DISCUSSION THREAD

**Discussion by paurray** [iterate-ch/cyberduck#15584] [Q&A] [answered]
Hello

I get the "enable Mountain duck finder extension" warning despite having it enabled.

On the finder level I can navigate to the folders on the Next Cloud Folder fine.

But there is a mountain duck yellow lamp ?warning? to the left of the connection I am attempting to make to Nextcloud via the Mountain Duck drop down menu up top of Mac OS Ventura 13.6.4.

But am also noticing other weirdness that I can not pin down.

Have also tried enabling/disabling multiple times “Mountain duck finder extension” and restarting. With random results eg disable it and it is enabled on restart. But I always get the warning. Regardless of what I do!

Is there something that I am missing?

thanks

Paul

edited and added version number: Version 4.15.3 (21820)

<img width="2083" alt="MountainDuckWeirdness WEB" src="https://github.com/iterate-ch/cyberduck/assets/9163424/013dd09c-661b-45db-a31b-8d95719c791d">

**Comment by dkocher** [comment, marked as answer]
Try to resume synchronization (currently paused as indicated by _Sync Paused_ by choosing _Resume Sync_ from the Mountain Duck sync status menu item in the status bar menu.

**Comment by paurray** [comment]
Thank you that seems to have done the trick. Silly of me to miss that.

**Comment by paurray** [comment]
Hi, hi

and sorry to return like this…
I think I spoke to soon.
I keep losing internet access maybe related/not related to this issue.
Which means I am doing a couple of restarts a day.
And then I get the same error message even after resyncing everything as suggested above.
Same screen shot this time I am synced and the there is a mountain duck green lamp to the left of the connection I am attempting to make to Nextcloud via the Mountain Duck drop down menu up top of Mac.
I also decided to try a fresh install of mountain duck using this here to uninstall:

https://freemacsoft.net/appcleaner/

At lest at face value what Appcleaner did looked kind of impressive cleaning out 26ish GBs of files and lots of folders.

But after a fresh install everything is back exactly the way it was.
Bookmarks etc are there.
Same sync job etc...

And I still have the same unexplained error.

Does not make much sense to me

thanks for any tips

Paul

<img width="1498" alt="MountainDuckWeirdness WEB-tk2" src="https://github.com/iterate-ch/cyberduck/assets/9163424/11ddbb12-bc51-44ec-8077-b20a7fcba9b4">

**Reply by tatze96** [reply]
> And I still have the same unexplained error.

Do you refer to the finder extension message, or do you also get other error messages?

Regarding the enable finder extension reminder:
We are aware of the frequent prompts to enable the Finder Extension but haven't found a solution yet.

As a workaround, choose *Don't ask again* before clicking on *Cancel* to deactivate the reminder.

Regarding the paused synchronization:
The synchronization can be paused manually but also gets paused by Mountain Duck if the connection to the server is lost or too unstable to transfer files. Files and folders that weren't indexed before the server or network connection dropped won't show up if you work offline.

In the case of Mountain Duck pausing the synchronization process due to connectivity errors, Mountain Duck performs reachability checks at regular intervals and will resume the synchronization automatically once the connection can be established again.

Regarding the reinstall:
Based on your description, it seems that the Appcleaner application didn't remove the [application support folder](https://docs.cyberduck.io/mountainduck/support/#application-support-folder) despite deleting around 26 GBs of files, as both the bookmarks and the synchronization queue are located in the application support folder.

**Comment by paurray** [comment]
Hello,

sorry my bad deleting the Mountain Duck App works.
I missed that I had to tick the check box in the App Cleaner App.
Which I guess is kind of cool as it was trying to protect me from myself.
I have now reinstalled and the only error I am still getting is the enable Mountain duck finder extension" warning despite having it enabled!
As advised I have clicked the "Don´t ask again" box
Would be nice if there was a fix sometime in the future :-)

<img width="613" alt="DeleteWorks-HumanError" src="https://github.com/iterate-ch/cyberduck/assets/9163424/5096c031-9249-4873-b0f9-ba70c354fa00">
